Formula/wownero.rb

```diff
@@ -0,0 +1,21 @@
+class Wownero < Formula
+  desc "Node and wallet binaries for the Wownero cryptocurrency project"
+  homepage "https://wownero.org"
+  url "https://git.wownero.com/attachments/56793f91-e5f1-4491-af96-516e53f6451d"
+  version "0.8.0.2"
+  sha256 "e5297223c28b3feb18704f0b0f7a242dd990d406da144abac4d1d0fb0317fb87"
+
+  bottle :unneeded
+
+  def install
+    bin.install "wownerod"
+    bin.install "wownero-wallet-cli"
+    bin.install "wownero-wallet-rpc"
+  end
+
+  test do
+    system "#{bin}/wownero-wallet-cli --version | grep 0.8.0.2"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I've refactored the test to actually utilize a built binary to validate it loads new wallet properly.